### DISCUSSION TITLE
firefox nightly 截图时选择范围后点击对勾图标后没有后续操作吗？

### DISCUSSION THREAD

**Discussion by yak93rise** [likaia/js-screen-shot#130] [Q&A] [answered]
看了剪贴板也没有，那个对勾是什么作用呢？

还是这是个bug，我不太确定。

**Comment by likaia** [comment, marked as answer]
> 看了剪贴板也没有，那个对勾是什么作用呢？
> 
> 还是这是个bug，我不太确定。

需要在https环境下运行，或者在localhost环境，文档有说明。
